tests/Tests/400__ods_dashboard/413__ods_dashboard_user_mgmt.robot

```diff
@@ -0,0 +1,86 @@
+    Switch Browser  1
+    Add OpenShift Groups To Data Science Administrators    rhods-noaccess
+    Add OpenShift Groups To Data Science User Groups       rhods-noaccess
+
+    AdminGroups In OdhDashboardConfig CRD Should Be        rhods-users
+    AllowedGroups In OdhDashboardConfig CRD Should Be      system:authenticated
+    Save Changes In User Management Setting
+    Page Should Contain  Unable to load User and group settings
+    Switch Browser  2
+    [Teardown]  Teardown For Admin UI   ${TEST_USER_3.USERNAME}   ${TEST_USER_3.PASSWORD}  ${TEST_USER_3.AUTH_TYPE}
+
+Verify Unauthorisez User Is Not Able To Spawn Jupyter Notebook
+    [Documentation]    Verify User Is Not Able To Spawn Jupyter Notebook
+    [Tags]   ODS-1680
+    ...     Tier1
+    ...     Sanity
+    Launch Dashboard And Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Clear User Management Settings
+    Add OpenShift groups To Data Science Administrators         rhods-admin
+    Add OpenShift Groups To Data Science User Groups   rhods-admin
+    Save Changes In User Management Setting
+    Close Browser
+    Launch Dashboard  ocp_user_name=${TEST_USER_3.USERNAME}  ocp_user_pw=${TEST_USER_3.PASSWORD}  ocp_user_auth_type=${TEST_USER_3.AUTH_TYPE}
+    ...               dashboard_url=${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  browser_options=${BROWSER.OPTIONS}
+    Menu.Navigate To Page    Applications    Enabled
+    Wait Until Page Contains    Launch application
+    Click Element               //*[@class="odh-card__footer__link"]
+    Wait Until Page Contains    Page Not Found   timeout=15
+    Page Should Contain         Page Not Found
+    [Teardown]  Teardown For Admin UI   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+
+Verify Automatically detects a group selected is removed and notify the user
+    [Documentation]  Verify if the group is deleted the user should get the
+     ...    message / notification
+    [Tags]  ODS-1686
+    ...     Tier1
+    ...     Sanity
+    Create Group  test
+    Launch Dashboard And Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Add OpenShift Groups To Data Science Administrators     test
+    Save Changes In User Management Setting
+    Delete Group  test
+    Reload Page
+    Wait Until Page Contains    Group no longer exist   timeout=20s
+
+*** Keywords ***
+Teardown For Admin UI
+    [Documentation]  Setup Default Values In User Management Settings
+    [Arguments]   ${username}  ${password}  ${auth_type}
+    Launch Dashboard And Check User Management Option Is Available For The User  ${username}  ${password}  ${auth_type}
+    Clear User Management Settings
+    Add OpenShift groups to Data Science administrators         dedicated-admins
+    Add OpenShift Groups To Data Science User Groups   system:authenticated
```

Comment:
you could use this kw to get the default groups to set: `Set Standard RHODS Groups Variables` and after that you can use `${STANDARD_ADMINS_GROUP}` and `${STANDARD_USERS_GROUP}` instead

Reply:
nope I don't think so we need to test the UI 
`Set Standard RHODS Groups Variables` will  use the cmd line approach

Reply:
agree added in teardown